key-rotator/manifest/manifest_test.go

```diff
@@ -227,6 +228,339 @@ func TestUpdateKeys(t *testing.T) {
 			}
 		})
 	}
+
+	// Failure tests.
+	for _, test := range []struct {
+		name       string
+		cfg        UpdateKeysConfig
+		wantErrStr string
+	}{
+		{
+			name: "empty batch signing key",
+			cfg: UpdateKeysConfig{
+				BatchSigningKey:             key.Key{},
+				BatchSigningKeyIDPrefix:     bskPrefix,
+				PacketEncryptionKey:         k(kv(0, k0)),
+				PacketEncryptionKeyIDPrefix: pekPrefix,
+				PacketEncryptionKeyCSRFQDN:  fqdn,
+			},
+			wantErrStr: "batch signing key has no key versions",
```

Comment:
Nit: Generally these, it seems like it'd be a bit nicer to reference them directly.

Reply:
Acknowledged; see my response to the other comment.